package io.knowit.backend.exception;

public class UserAlreadyExistsException extends RuntimeException {
}
